activities/services.go

```diff
@@ -0,0 +1,210 @@
+package activities
+
+import (
+	"clockify/helpers"
+	"clockify/models"
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"gorm.io/gorm"
+)
+
+type ActivitiesService struct {
+	db *gorm.DB
+}
+
+func NewActivitiesService(db *gorm.DB) *ActivitiesService {
+	return &ActivitiesService{
+		db: db,
+	}
+}
+
+func (s *ActivitiesService) GetAllActitives(w http.ResponseWriter, r *http.Request) {
```

Comment:
All of the Activities, Projects, Users service func should be behind a common interface